docs/button/button.md

```diff
@@ -15,6 +15,20 @@ import { CheckmarkIcon } from 'radiance-ui/lib/icons';
     <Button disabled>Disabled Button</Button>
     <Button isFullWidth>Full Width Button</Button>
 
+    <Button color="statusRed">Primary Color</Button>
```

Comment:
@daigof the problem with doing ${COLORS.statusRed} is that it gives the value, not the key. I need the key name like "statusRed" not "#abc123"

Reply:
ok, i changed the implementation to accept the color value instead. it makes the knobs dropdown work a little weird, but otherwise everything else is fine.

I will say that it feels a bit uncommon to pass a hex argument from an enumerated list rather than key names which is typical of other ui frameworks (semantic, bootstrap, foundation, material, etc)

For the sake of this pr i will make use of the `buttonColor={COLORS.statusRed}` but would def like to talk about the pros and cons offline of this.